src/views/signup/Index.jsx

```diff
@@ -32,23 +32,29 @@ export const SignUp = ({ signup, history }) => {
     username: '',
     email: '',
     password: '',
+    password2: '',
```

Comment:
Let's change `password2` to `confirmPassword`

Reply:
fixed